src/api/orders/types.ts

```diff
@@ -0,0 +1,46 @@
+import {LimitOrderV3Struct} from '../../limit-order'
+import {NetworkEnum} from '../../constants'
+
+export type OrdersApiConfig = {
+    network: NetworkEnum
+    url: string
+}
+
+export type ActiveOrdersRequestParams = {
+    page?: number
+    limit?: number
+}
```

Comment:
📓 in the gasless-swap they have default values, so it looks reasonable to make them optional